resources/certificate.rb

```diff
@@ -53,7 +55,7 @@
   else
     cmd = Mixlib::ShellOut.new("#{keytool} -list -keystore #{truststore} -storepass #{truststore_passwd} -v")
     cmd.run_command
-    Chef::Log.debug(cmd.format_for_exception)
+    Chef::Log.info(cmd.format_for_exception)
```

Comment:
will this show the password if it throws out?

Reply:
Possibly. 

Refactoring last. 👍